iam-password-policy-complexity/policy.rego

```diff
@@ -0,0 +1,137 @@
+package env0.policy
+
+# Check minimum password length
+deny[msg] {
+    r := input.plan.resource_changes[_]
```

Comment:
can'y you check that `create` or `update` are in `r.change.actions` instead of duplicating everything like this?

For all of course